Update shared-resources

LChernigovskaya opened the pull request:
diagram editor core is separated from robot specific code.

LChernigovskaya committed:
initial separation: in shared-resources there is core, in editor-service there is everything that is connected with robots(gestures, twoDModel)

jzuken commented on shared-resources/src/main/webapp/app/core/editorCore/model/RobotsDiagramNode.ts:
```diff
@@ -1,19 +1,3 @@
-/*
```
do we really need this element? how is it different form just DiagramNode?

jzuken replied:
I guess we'd better remove all robot-related code from here

jzuken commented on editor-service/src/main/webapp/app/robots/gestures/GesturesController.ts:
```diff
@@ -1,25 +1,8 @@
-/*
```
why are gestures in robots folder?

LChernigovskaya committed:
remove RobotsDiagramNode

remove RobotsDiagramNode from editor-core and add all images from
shared-resources to editor-service

LChernigovskaya committed:
some changes

1. Rename shared-resources to editor-core
2. Remove dependencies on editor-core in dashboard-service and
auth-service
3. Separate menu from editor(everything that is connected with creating,
saving and opening diagrams and folders)
4. Update grunt-ts
5. Return RootDiagranController.ts to connect editor with 2DModel

LChernigovskaya committed:
fix gestures

LChernigovskaya committed:
fix ports and links

jzuken commented:
I think we should remove `editor-core/out` from the repository since we generate them.

jzuken committed:
Remove generated js files

jzuken committed:
Remove some more obsolete copyright comments

jzuken committed:
Merge branch 'master' into editor-core
